### PR TITLE
Ash drake plushie!

### PR DESCRIPTION
**Pull request by Squi-RL**
Add: a cute little Ash drake plushie similar to the carp plushie
Imageadd: plush_drake icon, related icons for items_lefthand and righthand

### CODE REVIEW

**Comment by Poojawa** [comment]
You need to make sure you commit the .dmis as well see what Travis has outputted?


code/game/objects/items/toys.dm:1399:error: 'icons/obj/plush_drake.dmi': cannot find file

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1388,3 +1388,30 @@
...
+/obj/item/plush_drake
+	name = "Ash drake plushie"
+	desc = "An adorable stuffed toy that resembles a fierce dragon."
+	icon = 'icons/obj/plush_drake.dmi'
+	icon_state = "plush_drake"
+	item_state = "ash_plush"
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb = list("burned", "bitten", "swooped")
+	resistance_flags = FLAMMABLE
+	//var/bitesound = 'sound/weapons/bite.ogg'
```
Comment (Iamgoofball) [inline code/game/objects/items/toys.dm]: stop leaving useless code in

**Comment by Squi-RL** [comment]
sorry, still learning how this works

**Comment by Squi-RL** [comment]
right, I forgot to commit the changes last time, double checked that everything's in there so it should work... also removed the commented lines in the ash drake plush's code

**Comment by Poojawa** [comment]
While I'd still want a picture, I'm pretty sure you set it as your discord icon, so it's pretty cute.

an excellent first addition, well done.

**Pull request merged**